skyplane/config_paths.py

```diff
@@ -8,6 +8,7 @@
 aws_quota_path = __config_root__ / "aws_quota"
 azure_config_path = __config_root__ / "azure_config"
 azure_quota_path = __config_root__ / "azure_quota"
+azure_standardDv5_quota_path = __config_root__ / "azure_standardDv5_quota"
```

Comment:
Once we begin implementing the fall back logic for all VM types, we can coalesce them into a single file (azure_quota_path)